TRANSLATING.md

```diff
@@ -240,3 +261,24 @@ Some of our English page content is generated from outside sources, and must not
 However, these pages are translated directly here and **these warnings are not meant for translations**.
 
 For these generated pages (like `configuration-reference.md`), we recommend **ignoring and removing the note and component (including its import) from the file**, thus avoiding confusion for other translators thinking that this warning applies to translations as well.
+
+## Review Tips
+
+Reviewing PRs is also an important task — it's thanks to them that we can guarantee consistent, high-quality translations, so if you're interested in helping review translation PRs, thank you, we really appreciate the effort!
+
+**When you think a PR is good to be merged**, approve the PR through GitHub's "Review Changes" button or leave a "**LGTM!**" in the comments.
```

Comment:
Do you think it would be helpful to explain what LGTM means? (I half thought about mentioning our other fun versions of LGTM, but maybe in this guide it’s best to be clear and simple.)

```suggestion
**When you think a PR is good to be merged**, approve the PR through GitHub's "Review Changes" button or leave a "**LGTM!**" in the comments. (“LGTM” is an abbreviation of “Looks Good to Me” often used to approve pull requests.)
```

Reply:
I don't see why not. Sure could help someone that isn't familiarized with the abbreviation, good idea!